src/features/click-to-play.js

```diff
@@ -2374,6 +1350,10 @@ const updateHandlers = {
 export function init (args) {
     const websiteOwner = args?.site?.parentEntity
     const settings = args?.featureSettings?.clickToPlay || {}
+    const locale = args?.locale || 'en'
+    const localizedConfig = getConfig(locale)
```

Comment:
IMO the localisation shouldn't be mixed up with getConfig. I figure the config should just be data, with the string IDs. Then content-scope-scripts should provide a utility function that takes a string ID and the locale and returns the appropriate translation (that way other features could use it too). Then this script calls the translation function whenever it wants to turn one of the string IDs into the text to display to the user.

Reply:
I revisited this in https://app.asana.com/0/0/1203778992058963/1203923914388384/f, as it seems like a design decision that should be covered there?